app/pb_kits/playbook/pb_label_value/label_value.rb

```diff
@@ -2,55 +2,22 @@
 
 module Playbook
   module PbLabelValue
-    class LabelValue < Playbook::PbKit::Base
-      PROPS = %i[configured_classname
-                 configured_data
-                 configured_id
-                 configured_label
-                 configured_value].freeze
+    class LabelValue
+      include Playbook::Props
 
-      def initialize(classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     label: default_configuration,
-                     value: default_configuration)
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_label = label
-        self.configured_value = value
-      end
-
-      def label
-        if is_set? configured_label
-          pb_label = Playbook::PbCaption::Caption.new(text: configured_label)
-          ApplicationController.renderer.render(partial: pb_label, as: :object)
-        end
-      end
+      partial "pb_label_value/label_value"
 
-      def value
-        pb_body = Playbook::PbBody::Body.new do
-          default_value(configured_value, "")
-        end
-        ApplicationController.renderer.render(partial: pb_body, as: :object)
-      end
+      prop :label, type: Playbook::Props::String
+      prop :value, type: Playbook::Props::String,
+                   default: ""
 
-      def kit_class
-        "pb_label_value_kit"
+      def label?
+        label.present?
```

Comment:
Since `label` defaults to `nil`, which is `falsy`, there is not a need to make this presence checking method. `if object.label` would be fine to use in your template.